src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split('\n');
+  const filledLines = lines.filter(line => line.match(/[a-z]/));
+  const semicolonRemove = filledLines.map(line => line.slice(0, -1));
```

Comment:
You could split by `;` I think, but your solution is also ok

Reply:
Thank you for the tip) It is one full iteration shorter your way.
```suggestion
  const lines = sourceString.split(';');
  const filledLines = lines.filter(line => line.match(/[a-z]/));
```